### DISCUSSION TITLE
Running the jMusicBot.jar t launches Minecraft...?

### DISCUSSION THREAD

**Discussion by Lemur626** [jagrosh/MusicBot#1474] [Troubleshooting]
im trying to build a private discord music bot for my private server.  I followed the steps in this video on youtube video titles "Discord Music Bots are Dead…. Host your own!" by No Text To Speech. And after going step by step at the end when he launches the jMusicBot file its worked for him, but when i did it...it boots up Minecraft Launcher.... not sure what I did wrong or what may be causing this.

**Comment by MichailiK** [comment]
It's likely because you've (accidentally) associated `.jar` files with the Minecraft Launcher. Try right clicking JMusicBot.jar -> Open with -> Choose another app -> Java/OpenJDK -> Always.

If you can't find Java/OpenJDK in the list, try (re-)installing Java. The installer should hopefully automatically associate `.jar` files with Java for you.

![image](https://github.com/jagrosh/MusicBot/assets/39029839/a3a2246c-c696-46f9-9534-f31b4bdc7abb)

![image](https://github.com/jagrosh/MusicBot/assets/39029839/e72199b1-3317-4e1f-af1a-08452ab5a8c0)
